resources/assets/lib/beatmapsets-show/metadata-editor.tsx

```diff
@@ -135,6 +152,7 @@ export default class MetadataEditor extends React.PureComponent<Props, State> {
         genre_id: this.state.genreId,
         language_id: this.state.languageId,
         nsfw: this.state.nsfw,
+        offset: isNaN(Number(this.state.offset)) ? 0 : this.state.offset,
```

Comment:
```suggestion
        offset: isNaN(Number(this.state.offset)) ? undefined : this.state.offset,
```
so it doesn't reset an existing valid offset